docs/SUMMARY.md

```diff
@@ -21,11 +21,13 @@
 
 - [Policy engine](./topics/policy.md)
 - [Authorization and sessions](./topics/authorization.md)
+- [Use the Admin API](./topics/admin-api.md)
 
 # Reference
 
 - [Configuration file reference](./reference/configuration.md)
 - [GraphQL API](./reference/graphql.md)
```

Comment:
We should rename the GraphQL API to be Internal GraphQL API. Some things to do:

- Move the existing GraphQL API documentation into the Development section of the documentation hierachy.
- Change all references to "GraphQL API" in the documentation to "Internal GraphQL API".
- Add a callout section at the top of the Internal GraphQL API documentation to say:
  - This API is now for internal use only
  - External usage is now deprecated and will be blocked/removed in a future release
  - The new Admin API should be used instead

Reply:
I've done this myself.